scripts/build.sh

```diff
@@ -1,44 +1,49 @@
-#!/bin/bash -e
+#!/bin/bash
+# File managed by pluginsync
 
-#http://www.apache.org/licenses/LICENSE-2.0.txt
+# http://www.apache.org/licenses/LICENSE-2.0.txt
 #
 #
-#Copyright 2015 Intel Corporation
+# Copyright 2016 Intel Corporation
 #
-#Licensed under the Apache License, Version 2.0 (the "License");
-#you may not use this file except in compliance with the License.
-#You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-#    http://www.apache.org/licenses/LICENSE-2.0
+#     http://www.apache.org/licenses/LICENSE-2.0
 #
-#Unless required by applicable law or agreed to in writing, software
-#distributed under the License is distributed on an "AS IS" BASIS,
-#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#See the License for the specific language governing permissions and
-#limitations under the License.
-
-GITVERSION=`git describe --always`
-SOURCEDIR=$1
-BUILDDIR=$SOURCEDIR/build
-PLUGIN=`echo $SOURCEDIR | grep -oh "snap-.*"`
-ROOTFS=$BUILDDIR/rootfs
-BUILDCMD='go build -a -ldflags "-w"'
-
-echo
-echo "****  Snap Plugin Build  ****"
-echo
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-# Disable CGO for builds
-export CGO_ENABLED=1
+set -e
+set -u
+set -o pipefail
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__proj_dir="$(dirname "$__dir")"
+
+# shellcheck source=scripts/common.sh
+. "${__dir}/common.sh"
 
-# Clean build bin dir
-rm -rf $ROOTFS/*
+plugin_name=${__proj_dir##*/}
+build_dir="${__proj_dir}/build"
+go_build=(go build -ldflags "-w")
 
-# Make dir
-mkdir -p $ROOTFS
+_info "project path: ${__proj_dir}"
+_info "plugin name: ${plugin_name}"
+
+# Disable CGO for builds
+export CGO_ENABLED=0
```

Comment:
Cody saw this too, I think it's a mistake because it's not in sbr and it builds correctly.